transformers/events/vat_init/converter.go

```diff
@@ -19,38 +19,42 @@ package vat_init
 import (
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
+	"github.com/makerdao/vulcanizedb/libraries/shared/factories/event"
 	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 )
 
-type VatInitConverter struct{}
+type Converter struct{}
 
 const (
 	logDataRequired   = false
 	numTopicsRequired = 2
```

Comment:
We can use the [shared converter constants ](https://github.com/makerdao/vdb-mcd-transformers/blob/staging/transformers/shared/converter.go#L32) for `logDataRequired` and then perhaps we can add a new constants for `TwoTopicsRequired`.